spring/boot3-autoconfigure/src/main/java/com/linecorp/armeria/spring/ArmeriaSettings.java

```diff
@@ -430,6 +430,12 @@ public void setFormat(@Nullable String format) {
      */
     private List<Port> ports = new ArrayList<>();
 
+    /**
+     * The context path to serve the requests on. If not set, requests will be served on the root context path.
+     */
+    @Nullable
+    private String contextPath;
```

Comment:
nit; what do you think of the naming `baseContextPath` to match with the API at `ServerBuilder`?

Reply:
`context-path` is used since Spring users may try to find context- first and IDE will autocomplete for that. https://www.baeldung.com/spring-boot-context-path#1-usingapplicationproperties-yml
But I wasn't strong here. Let me listen to other folk's opinions as well before going further.

Reply:
If auto-completion is the goal, `spring-boot-configuration-processor` is probably sufficient.

Having said this, if `context-path` works in the same way as spring I guess I'm fine with the current naming